src/core/algorithm/timelinePagerank.js

```diff
@@ -204,28 +205,47 @@ export async function _computeTimelineDistribution(
     const nodeToConnections = NullUtil.get(
       nodeToConnectionsIterator.next().value
     );
-    const {chain} = createOrderedSparseMarkovChain(nodeToConnections);
-
-    const seed = weightedDistribution(nodeOrder, nodeWeights);
-    if (pi0 == null) {
-      pi0 = seed;
-    }
-    const params: PagerankParams = {chain, alpha, seed, pi0};
-    const distributionResult = await findStationaryDistribution(params, {
-      verbose: false,
-      convergenceThreshold: 1e-7,
-      maxIterations: 255,
-      yieldAfterMs: 30,
-    });
-    const intervalWeight = sum(nodeWeights.values());
-    results.push({
+    const result = await _intervalResult(
+      nodeWeights,
+      nodeToConnections,
+      nodeOrder,
       interval,
-      intervalWeight,
-      distribution: distributionResult.pi,
-    });
+      pi0,
+      alpha
+    );
+    results.push(result);
     // Use the latest convergce results as the starting point for the next run
     // of PageRank
-    pi0 = distributionResult.pi;
+    pi0 = result.distribution;
   }
   return results;
 }
+
+export async function _intervalResult(
+  nodeWeights: Map<NodeAddressT, number>,
+  nodeToConnections: NodeToConnections,
+  nodeOrder: $ReadOnlyArray<NodeAddressT>,
+  interval: Interval,
+  pi0: Distribution | null,
+  alpha: number
+): Promise<IntervalResult> {
+  const {chain} = createOrderedSparseMarkovChain(nodeToConnections);
+
+  const seed = weightedDistribution(nodeOrder, nodeWeights);
+  if (pi0 == null) {
+    pi0 = seed;
+  }
+  const params: PagerankParams = {chain, alpha, seed, pi0};
```

Comment:
Nit: prefer not to reassign function arguments. (wish this had an eslint rule, couldn't find one though)
Instead, can use something like:

```js
const params: PagerankParams = {
  chain,
  alpha,
  seed,
  pi0: pi0 || seed
};
```